### DISCUSSION TITLE
Wallpaper

### DISCUSSION THREAD

**Discussion by henrikmagni** [ErikReider/SwayNotificationCenter#488] [Q&A] [answered]
Does anyone know which wallpaper is used in the readme (linked below), and where to get it? I tried doing a reverse image search, but all I found was the wallpaper being used with product pictures of a Linux tablet called "Juno Tab 3".

https://github.com/ErikReider/SwayNotificationCenter/blob/main/assets/panel.png

**Comment by ErikReider** [comment, marked as answer]
Here: https://gitlab.gnome.org/GNOME/gnome-backgrounds/-/blob/gnome-45/backgrounds/adwaita-l.jpg?ref_type=heads

It's the stock Gnome 45 Adwaita wallpaper :)
